src/proxy-ui-api/src/test/java/org/niis/xroad/securityserver/restapi/util/TestUtils.java

```diff
@@ -365,4 +368,27 @@ public static void mockServletRequestAttributes() {
         ServletRequestAttributes attributes = new ServletRequestAttributes(request);
         RequestContextHolder.setRequestAttributes(attributes);
     }
+
+    /**
+     * Checks if the sort order of the given Set is correct.
+     * @param set
+     * @param comparator
+     * @return
+     */
+    public static boolean isSortOrderCorrect(Set set, Comparator comparator) {
```

Comment:
Raw types should not be used, consider
`public static <T> boolean isSortOrderCorrect(Set<T> set, Comparator<? super T> comparator)`
(requires also other changes)
And since we have Guava as a dependency, this function can be simplified to `Ordering.from(comparator).isOrdered(set)`

Reply:
I removed the custom logic and used Guava instead. Also, the method signature was updated.